programming/tools/android-tools/pspec.xml

```diff
@@ -28,6 +28,14 @@
         </Files>
     </Package>
     <History>
+        <Update release="1">
```

Comment:
Please change the release number to "7" here when you create the pull request at the new repository (this number increments with every package update).